ipn/prefs.go

```diff
@@ -37,6 +37,9 @@ type Prefs struct {
 	// If empty, the default for new installs, DefaultControlURL
 	// is used. It's set non-empty once the daemon has been started
 	// for the first time.
+	//
+	// TODO(apenwarr): Note: this value can't be updated using SetPrefs().
```

Comment:
What's the TODO? The verb tense isn't clear. To make SetPrefs let you change this at runtime? Rewrite the TODO with an imperative.

Reply:
Fixed.